src/wallet/db.cpp

```diff
@@ -826,6 +826,10 @@ void BerkeleyDatabase::Flush(bool shutdown)
             LOCK(cs_db);
             g_dbenvs.erase(env->Directory().string());
             env = nullptr;
+        } else {
+            // To avoid accessing a map that has already deconstructed, do not call this when shutdown is true. g_dbenvs.erase would also erase this value.
+            // TODO: get rid of wild pointers
```

Comment:
what are "wild pointers"?

Reply:
This is based on #14559, should comment there.

Reply:
Actually it's #14320

Reply:
aka prchain.